src/bot/config.py

```diff
@@ -66,3 +66,13 @@ def getenv(name: str, astype: Type[T] = str, default: T = D, required: bool = Fa
     # WSS provider seems to be broken in python 3.10 and
     # doesn't work in the current flow. Magic asyncio fails happen.
     raise RuntimeError("Only http[s] Web3 provider endpoint supported")
+
+if os.getenv("HTTP_REQUESTS_RETRY"):
+    HTTP_REQUESTS_RETRY = int(os.getenv("HTTP_REQUESTS_RETRY"))
+else:
+    HTTP_REQUESTS_RETRY = 3
```

Comment:
```suggestion
HTTP_REQUESTS_RETRY = getenv("HTTP_REQUESTS_RETRY", int, default=3)
```

and the same for the one below